### PR TITLE
CNV#36159: RN - new configuration option SSH over NodePort service

### PR DESCRIPTION
**Pull request by danielclowers**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/CNV-36159

Link to docs preview:
https://71714--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes#virt-4-15-web

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 1]
```diff
@@ -111,6 +111,7 @@ This release adds new features and enhancements related to the following compone
 //CNV-36162: Search option in VM configuration tab
 
 //CNV-36159: New configuration option "SSH over NodePort service"
+* A virtual machine's configuration section now allows *SSH over NodePort* as a service type.
```
Comment (gouyang) [inline virt/release_notes/virt-4-15-release-notes.adoc:114]: Can it be a bit more specific about the configuration of SSH over Nodeport?
Maybe should rephase it like the item below?

**Review by skopacz1 — APPROVED** [review]
LGTM

**Pull request merged**